### PR TITLE
SAK-43343 Forums/Rubrics use corresponding forum/topic rubric association

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java [hunk 1]
```diff
@@ -9212,18 +9212,16 @@ public boolean hasAssociatedRubric(){
...
-		}
-	}
+    if (gradeAssign != null && rubricsService.hasAssociatedRubric(RubricsConstants.RBCS_TOOL_GRADEBOOKNG, gradeAssign)) {
+      return gradeAssign;
+    }
+    return null;
+  }
 
-	public String getCDNQuery() {
+  public String getCDNQuery() {
```
Comment (bgarciaentornos) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java]: Indentation

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java [hunk 2]
```diff
@@ -9212,18 +9212,16 @@ public boolean hasAssociatedRubric(){
 		return (allowedToGradeItem && (getRubricAssociationId() != null));
 	}
 
-	public String getRubricAssociationId(){
+  public String getRubricAssociationId() {
+    String gradeAssign = selectedTopic != null ? selectedTopic.getGradeAssign() : selectedForum != null ? selectedForum.getGradeAssign() : null;
```
Comment (bgarciaentornos) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java]: Maybe adding parenthesis for readability?

Comment (bgarciaentornos) [reply msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java]: Indentation is off too

**Comment by ern** [comment]
Here is a reference to lit-html binding types
https://lit-html.polymer-project.org/guide/template-reference#binding-types

**Pull request merged**